complete/_rg

```diff
@@ -167,7 +167,7 @@ _rg() {
 
     + '(pcre2-unicode)' # PCRE2 Unicode options
     $no'(--no-pcre2-unicode)--pcre2-unicode[enable PCRE2 Unicode mode (with -P)]'
-    '(--no-pcre2-unicode)--no-pcre2-unicode[disable PCRE2 Unicode mode (with -P)]'
+    '(--pcre2-unicode)--no-pcre2-unicode[disable PCRE2 Unicode mode (with -P)]'
```

Comment:
cc @okdana I made this change based on your docs. Let me know if I got this wrong!

Reply:
Oh, i was the one who got it wrong actually, sorry. :/

Neither of those exclusions are necessary since we've got the parentheses around the group name, but i *had* intended to exclude `--no-pcre2` (because there's no reason you'd change the PCRE2 Unicode mode if you were just going to disable PCRE2 anyway). I guess i just wasn't paying attention.

So both of the options in that group should actually have `(--no-pcre2)`.